src/rgw/const.py

```diff
@@ -141,6 +141,24 @@
 UTF_ENCODING = 'utf-8'
 MOTR_MY_FID = f'{SVC_SECTION}>motr my fid'
 
+# RGW resource limit keys
+SVC_CPU_MIN_VAL_LIMIT = '250m'
```

Comment:
these limit values are currently copied from existing solution.yaml file.
Saurabh is exploring on exact values in current sprint and we will update these values in Sprint 75 with more specific values.

Reply:
@sachitanands , are supporting cpu limit values without any unit specified by user? for example cpu : 1 is also a valid value.

Reply:
yes. added a logic to handle this numeric value. (e.g. 1 cpu = 1000m) any fractional value will also be handled in similar way.